src/Testing/Application.php

```diff
@@ -90,7 +90,7 @@ public function __construct(?string $basePath = null)
      *
      * @return string
      */
-    public function basePath($path = ''): string
+    public function basePath(string $path = ''): string
```

Comment:
I believe this will break Laravel 6

Reply:
This is the linting issue that was breaking the master build.